python/ray/tune/schedulers/pbt.py

```diff
@@ -309,8 +312,10 @@ def _exploit(self, trial_executor, trial, trial_to_clone):
 
         new_tag = make_experiment_tag(trial_state.orig_tag, new_config,
                                       self._hyperparam_mutations)
-        reset_successful = trial_executor.reset_trial(trial, new_config,
-                                                      new_tag)
+        trial.reset_logger()
+        new_config["new_logdir"] = trial.logdir
```

Comment:
is this line needed?

Reply:
Oups, those 2 lines should not be there.